sdk/servicebus/service-bus/src/models.ts

```diff
@@ -105,8 +105,7 @@ export interface ServiceBusReceiverOptions {
 
   /**
    * The maximum duration in milliseconds until which the lock on the message will be renewed
-   * by the sdk automatically. This auto renewal stops once the message is settled or once the user
-   * provided onMessage handler completes ite execution.
+   * by the sdk automatically. This auto renewal stops once the message is settled.
```

Comment:
(or expires, but that's obvious, I think)

Reply:
Yeah, I don't think it needs a mention!